### PR TITLE
Sandy mobs

### PR DESCRIPTION
**Pull request by Superyodeler**
## About The Pull Request

Contains the changes Zeta wanted to the mobs. Everything worked on my machine, but I may have had a slight headache at some point while working on this so be sure to double check stuff.

## Why It's Good For The Game

Zeda mobs good

## Changelog
:cl:
tweak: Added zeda's mob changes
/:cl:

### CODE REVIEW

File: _maps/_basemap.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-//#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
+#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
```
Comment (Maldaris) [inline _maps/_basemap.dm]: Comment this back out.

File: code/modules/mob/living/living_defense.dm [hunk 2]
```diff
@@ -48,6 +49,7 @@
 	return BULLET_ACT_HIT
 
 /mob/living/bullet_act(obj/projectile/P, def_zone)
+	message_admins("Bullet_act")
```
Comment (Maldaris) [inline code/modules/mob/living/living_defense.dm]: debug messages should be removed

File: code/modules/mob/living/living_defense.dm [hunk 3]
```diff
@@ -1,5 +1,6 @@
 
 /mob/living/proc/run_armor_check(def_zone = null, attack_flag = "melee", absorb_text = null, soften_text = null, silent=FALSE, armour_penetration, penetrated_text)
+	message_admins("run_armor_check")
```
Comment (Maldaris) [inline code/modules/mob/living/living_defense.dm]: debug messages should be removed

File: code/modules/mob/living/simple_animal/animal_defense.dm [hunk 4]
```diff
@@ -108,11 +108,14 @@
 	else
 		apply_damage(damage, damagetype, null, getarmor(null, armorcheck))
 		return TRUE
-
+/*
```
Comment (Maldaris) [inline code/modules/mob/living/simple_animal/animal_defense.dm]: If we're commenting out things, we should just remove them.

Comment (Superyodeler) [reply code/modules/mob/living/simple_animal/animal_defense.dm]: This is the way we have been handling modular removals so far.

**Review by MarkSuckerberg — APPROVED** [review]

**Pull request merged**